examples/mp-faulttolerance-fallback/README_es.adoc

```diff
@@ -0,0 +1,125 @@
+= MicroProfile Fault Tolerance - Fallback
+:index-group: MicroProfile
+:jbake-type: page
+:jbake-status: published
+
+Este es un ejemplo de como usar Microprofile @Fallback en TomEE.
+
+== Fallback Feature
+Fault Tolerance Fallback provee una alternativa en caso de fallo de una ejecución. Esta alternativa será llamada cuando 
+un `Retry` o `CircuitBreaker` ha fallado.
+
+Para usar esta funcionalidad es necesario anotar el metodo con  `@Fallback`.
+
+Las politicas del Fallback permiten configurar :
+
+* **value**: Una clase que implementa `FallbackHandler`
+* **fallbackMethod**: El método que será ejecutado.
+
+Los parámetros `value` and `fallbackMethod` no pueden ser específicados a la vez.
+
+Para más detalles revisar http://download.eclipse.org/microprofile/microprofile-fault-tolerance-1.1/microprofile-fault-tolerance-spec.html[Especificación] 
+
+== Ejemplpos
```

Comment:
HI @gdiazs,
please fix this name .

thanks.

Reply:
Thanks